tools/network-scene-vis.md

```diff
@@ -0,0 +1,61 @@
+---
+id: netscenevis
+title: Network Scene Visualization
+---
+
+The Network Scene Visualization (NetSceneVis) is a powerful, user-friendly tool included in the Multiplayer Tools package to help you visualize and debug network communication on a per-object basis in the scene view of your project with visualizations such as mesh shading and text overlay.
+
+When a game session is started, network messages contain information about the game state, such as the position of game objects and player actions. The NetSceneVis tool intercepts these network messages to visually display them in the Unity Editor. Each player is represented by a colored sphere, and lines show the network messages being sent between the players. The state of each GameObject in the scene, including their position, rotation, and any other relevant data is also shown.
+
+The visualizations are updated in real-time as the network messages are sent and received, showing the game state synchronization across the network.
+
+NetSceneVis can help diagnose issues related to network latency, packet loss, and other network-related issues that may affect the player experience in your multiplayer games. This tool can help optimize your network code to ensure that your game runs smoothly for all players.
```

Comment:
I think that NetSceneVisualization would be useful for examining bandwidth and ownership on a per-object basis, but I don't think this would help to diagnose issues related to network latency, and packet loss 🤔

The last sentence is probably true though, as the samples team has found it useful in optimizing their networking and eliminating needless network traffic and bandwidth from idle objects.